doc/releases/changelog-dev.md

```diff
@@ -179,6 +179,10 @@
 * `Controlled` operators now work with `qml.is_commuting`.
   [(#2994)](https://github.com/PennyLaneAI/pennylane/pull/2994)
 
+* The `expand_matrix()` has been moved from `~/operation.py` to 
```

Comment:
I think we should put this under "Breaking change", just in case someone relies on this function.